docker-compose.yml

```diff
@@ -2,8 +2,8 @@ version: "2.4"
 services:
   db:
     image: postgres:13.2-alpine
-    ports:
-      - "5432:5432"
+    expose:
+      - "5432"
```

Comment:
I think some folks on occasion do connect to the local DB using psql. It'd be nice to leave this one mapped to the host as well.

Reply:
I have never done this but that shouldn't prevent anyone else from being able to do so. Exposed port in d714b04.